packages/sdks/typescript/src/utils/nodeFetchResponseToReadableStream.ts

```diff
@@ -43,7 +46,14 @@ export function nodeFetchResponseToReadableStream(nodeStream: Readable) {
        * an underlying internal failure, or when a stream implementation attempts to push an invalid chunk of data.
        */
       nodeStream.on('error', (err) => {
-        controller.error(err)
+        try {
+          controller.error(err)
+        } catch (e) {
+          // controller might be claused already
+          if (onError) {
```

Comment:
So this avoid the issue or just improve error message?

Reply:
This will report the error correctly even if the controller is already closed